### ISSUE TITLE
deactivate intellisense when in comment mode

### ISSUE THREAD

**Issue by phlavenk** [julia-vscode/julia-vscode#1094]
When using `Literate.jl` or `Weave` a lot of text is written into comment sections of a .jl document. When writing in plain english, the intellisense word completion of the last word in a row gets accepted - even if the user just wanted to continue on next line. 
Is it possible to inhibit code-completion when entering a comment string  - after `#` or `#= ... =#`?

**Comment by CameronBieganek** [comment]
VS Code provides options that you can set to control this behavior. You can completely turn off suggestions in comments with this in your "settings.json" file:

```json
{
    "editor.quickSuggestions": {
        "comments": false
    }
}
```

Or you can configure suggestions to not accept a suggestion on enter (and thus only accept suggestions on hitting the tab key):

```json
{"editor.acceptSuggestionOnEnter": "off"}
```

**Comment by phlavenk** [comment]
I've added this knowledge into wiki-workarounds section.
Closing as resolved, thanks @CameronBieganek.
